app/models/ecf_participant_eligibility.rb

```diff
@@ -35,7 +35,7 @@ def determine_status
                                    %i[ineligible previous_participation]
                                  elsif previous_induction? && participant_profile.ect?
                                    %i[ineligible previous_induction]
-                                 elsif !qts?
+                                 elsif !qts? && participant_profile.participant_type != :mentor
```

Comment:
Just a suggestion - neither read particularly nicely
```suggestion
                                 elsif !qts? && !participant_profile.mentor?
```